.github/workflows/lint.yml

```diff
@@ -31,16 +31,16 @@ jobs:
 
   # hadolint lints the Dockerfile
   hadolint:
-    uses: celestiaorg/.github/.github/workflows/reusable_dockerfile_lint.yml@main # yamllint disable-line rule:line-length
```

Comment:
[Optiona] To make it easier to update the version value in the future, we may define a constant for the version number and reference it throughout the file instead of hard-coding the version number in multiple places.

Reply:
yeah, why do we do this for all of our github actions? do github actions allow for such a thing? that would be nice since I couldn't find an instance where we need different versions